pmlc/dialect/pxa/transforms/BUILD

```diff
@@ -32,3 +36,10 @@ plaidml_cc_library(
     deps = [":lib"],
     alwayslink = 1,
 )
+
+heatmap(
```

Comment:
I'd prefer to keep all of the x86 specific things over in `pmlc/target/x86`

Reply:
OK. I had no idea what the overall structure is. I thought the xsmm will be use by this very specific XSMM/CPU stenciler. So I put it there.
I will move it.

Reply:
Also, in the auto_stenciler, (the old code base) it is done the same way. This is just a call to a function from a bzl file, I think.